### ISSUE TITLE
The ColorPicker component does not hide some of  its children

### ISSUE THREAD

**Issue by AchrafBn** [hxf31891/react-gradient-color-picker#103]
Hello,

I'm facing  an issue with hiding some of the children in the [Controls](https://github.com/hxf31891/react-gradient-color-picker/blob/main/src/components/Controls.tsx#L124) component. When I set the boolean props hideColorGuide, hideAdvancedSliders, and hideInputType in my component, those elements are not removed from the DOM
### Basic code

```
function App() {

	const [color, setColor] = useState("rgba(255,255,255,1)");

	return (
		<div>
			<h2>Color picker</h2>
			<ColorPicker
				value={color}
				onChange={setColor}
				hideColorGuide
				hideAdvancedSliders
				hideInputType
			/>
		</div>
	);
}
```
### Environment

NodeJs: v20.12.2
Vite: v5.0.8
react: v18.2.0
react-best-gradient-color-picker: v3.0.10,

**Comment by chrioniep** [comment]
I have the same problem

**Comment by hxf31891** [comment]
@chrioniep @AchrafBn this should be resolved in v3.0.11
